docs/user-guide/rug/editors.md

```diff
@@ -4,44 +4,380 @@ resources.
 
 Rug editors automate updates, from the simplest to most complex.
 
+## Anatomy of an Editor
+
+Assume our team has an existing Spring Boot Rest project cloned or copied/pasted
+from in other projects. These manual actions are brittle at best and usually
+ends up with team members wondering what they did wrong. 
+
+Let's suppose however that project is already a Rug project, it would still
+be a running Spring Boot project in its own right. Here is such a project:
```

Comment:
I'm still not sold that we should be introducing editors intermixed with working projects. Why not introduce them using the more common case: a Rug project containing editors that can run on all your projects?